src/documentation/explore/ToolkitTopLevelListItem.tsx

```diff
@@ -15,20 +15,23 @@ import {
   ListItemProps,
   ListItem,
   Divider,
+  Tag,
 } from "@chakra-ui/react";
 import { imageUrlBuilder } from "../../common/imageUrlBuilder";
 
 interface ToolkitTopLevelListItemProps {
   name: string;
   description: ReactNode;
   icon?: ToolkitImage;
+  isV2Only?: Boolean;
```

Comment:
```suggestion
  isV2Only?: boolean;
```

https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#number-string-boolean-symbol-and-object

Reply:
[`8cc14ff2ff`](https://github.com/microbit-foundation/python-editor-next/pull/443/commits/8cc14ff2fff445b903115f94c87ea845d7f71e49)